docs/reference/configuration.md

```diff
@@ -168,6 +168,8 @@ This reference uses `.` to denote the nesting of values.
 * `gateways.s3.region` `(string : "us-east-1")` - AWS region we're pretending to be in, it should match the region configuration used in AWS SDK clients
 * `gateways.s3.fallback_url` `(string)` - If specified, requests with a non-existing repository will be forwarded to this URL. This can be useful for using lakeFS side-by-side with S3, with the URL pointing at an [S3Proxy](https://github.com/gaul/s3proxy) instance.
 * `stats.enabled` `(bool : true)` - Whether to periodically collect anonymous usage statistics
+* `stats.flush_interval` `(duration : 30s)` - Interval used to post anonymous statistics collected
+* `stats.flush_size` `(int : 100)` - A size of anonymous statistics collected in which we post
```

Comment:
Size in what? Records / MBs?

Reply:
Records - each event include a key (composite of the events fields) and count

Reply:
Added "records" to the doc line